frontend/src/lib/components/KeyboardHotkeys.tsx

```diff
@@ -0,0 +1,64 @@
+import { useEventListener } from 'lib/hooks/useEventListener'
+import React from 'react'
+import { Keys } from '~/types'
+
+interface HotKeyInterface {
+    action: () => void
+    disabled?: boolean
+}
+
+interface HotKeys {
+    a?: HotKeyInterface
```

Comment:
@mariusandra would love your input here. As you can see the keys are being repeated manually but are already registered as a type `Keys` on `~/types.ts`, is there a way to use that type here without having to repeat everything?

Reply:
Indeed there is :) Take a look now

Reply:
awesome! thanks Michael! so simple in hindsight 🙈